includes/Export.php

```diff
@@ -995,65 +1242,110 @@ function closeAndRename( $newname, $open = false ) {
 	}
 }
 
-
-
 /**
  * Dump output filter class.
  * This just does output filtering and streaming; XML formatting is done
  * higher up, so be careful in what you do.
  * @ingroup Dump
  */
 class DumpFilter {
+	/**
+	 * @var DumpOutput
+	 * FIXME will need to be made protected whenever legacy code
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this "FIXME" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)